src/components/gcds-button/gcds-button.css

```diff
@@ -14,233 +18,137 @@
   box-shadow: var(--custom-gcds-style-box-shadow);
   text-transform: var(--custom-gcds-style-text-transform);
 
-  &.primary {
-    &.solid {
-      background-color: var(--custom-gcds-style-background-color, var(--gcds-colour-base-blue-grey-900));
-      color: var(--gcds-colour-utils-white);
+  transition: all .15s ease-in-out;
 
-      &:hover, &.hover {
-        
-      }
-      &:focus, &.focus {
-        
-      }
-      &:active, &.active {
-        
-      }
-      &:disabled, &.disabled {
-        
-      }
-    }
-    &.outline {
-
-      &:hover, &.hover {
-        
-      }
-      &:focus, &.focus {
-        
-      }
-      &:active, &.active {
-        
-      }
-      &:disabled, &.disabled {
-        
-      }
-    }
-    &.text-only {
-
-      &:hover, &.hover {
-        
-      }
-      &:focus, &.focus {
-        
-      }
-      &:active, &.active {
-        
-      }
-      &:disabled, &.disabled {
-        
-      }
-    }
+  /* Shared Styles */
+  &[aria-disabled="true"] {
```

Comment:
Should we add a `cursor: not-allowed` to the disabled state?

Reply:
Good call, I just added it in.